Modules/_opcode.c

```diff
@@ -67,30 +67,34 @@ _opcode_stack_effect_impl(PyObject *module, int opcode, PyObject *oparg,
     return effect;
 }
 
-
-
-
 static PyMethodDef
 opcode_functions[] =  {
     _OPCODE_STACK_EFFECT_METHODDEF
     {NULL, NULL, 0, NULL}
 };
 
+static int
+_opcode_exec(PyObject *m)
```

Comment:
I don't see the point of an empty exec function. You can just remove it, no?

Reply:
@vstinner It's necessary for pep489, no?

Reply:
module init will judge the slots' value. module init will skip exec the slots when slots just only have `{0, NULL}`.
You could take a look the code when you have free time ; )
[PyModule_ExecDef](https://github.com/python/cpython/blob/master/Objects/moduleobject.c#L393)

Reply:
> @vstinner It's necessary for pep489, no?

No.